src/BizHawk.Emulation.Cores/Consoles/3DO/Opera.cs

```diff
@@ -0,0 +1,324 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using BizHawk.Emulation.Common;
+using BizHawk.Emulation.Cores;
+using BizHawk.Emulation.Cores.Waterbox;
+using BizHawk.Emulation.DiscSystem;
+
+namespace BizHawk.Emulation.Consoles.ThreeDO
+{
+	[PortedCore(
+		name: CoreNames.Opera,
+		author: "Libretro Team",
+		portedVersion: "2025.03.08 (67a29e6)",
+		portedUrl: "https://github.com/libretro/opera-libretro",
+		isReleased: false)]
+	public partial class Opera : WaterboxCore
+	{
+		private static readonly Configuration ConfigNTSC = new Configuration
+		{
+			SystemId = VSystemID.Raw._3DO,
+			MaxSamples = 8 * 1024,
+			DefaultWidth = LibOpera.NTSC_WIDTH,
+			DefaultHeight = LibOpera.NTSC_HEIGHT,
+			MaxWidth = LibOpera.PAL2_WIDTH,
+			MaxHeight = LibOpera.PAL2_HEIGHT,
+			DefaultFpsNumerator = LibOpera.NTSC_VIDEO_NUMERATOR,
+			DefaultFpsDenominator = LibOpera.NTSC_VIDEO_DENOMINATOR,
+		};
+
+		private static readonly Configuration ConfigPAL1 = new Configuration
+		{
+			SystemId = VSystemID.Raw._3DO,
+			MaxSamples = 8 * 1024,
+			DefaultWidth = LibOpera.PAL1_WIDTH,
+			DefaultHeight = LibOpera.PAL1_HEIGHT,
+			MaxWidth = LibOpera.PAL1_WIDTH,
+			MaxHeight = LibOpera.PAL1_HEIGHT,
+			DefaultFpsNumerator = LibOpera.PAL1_VIDEO_NUMERATOR,
+			DefaultFpsDenominator = LibOpera.PAL1_VIDEO_DENOMINATOR,
+		};
+
+		private static readonly Configuration ConfigPAL2 = new Configuration
+		{
+			SystemId = VSystemID.Raw._3DO,
+			MaxSamples = 8 * 1024,
+			DefaultWidth = LibOpera.PAL2_WIDTH,
+			DefaultHeight = LibOpera.PAL2_HEIGHT,
+			MaxWidth = LibOpera.PAL2_WIDTH,
+			MaxHeight = LibOpera.PAL2_HEIGHT,
+			DefaultFpsNumerator = LibOpera.PAL2_VIDEO_NUMERATOR,
+			DefaultFpsDenominator = LibOpera.PAL2_VIDEO_DENOMINATOR,
+		};
+
+		private readonly List<IDiscAsset> _discAssets;
+
+		private string GetFullName(IRomAsset rom) => rom.Game.Name + rom.Extension;

```

Comment:
Ditto of https://github.com/TASEmulators/BizHawk/pull/4217#discussion_r1999430713

Reply:
Removed this function. Wasn't called anywhere